### DISCUSSION TITLE
2 Dates Spanning Time Release?

### DISCUSSION THREAD

**Discussion by litifyse** [deejay-hub/timeline-lwc#245] [Q&A] [answered]
Hey! Love this app.

Reading The FAQ Knowledge Base it mentions the below:

Can I plot a record that spans a period of time?

Not yet. But this change has already been suggested and implemented so we just need time to test it and we will endeavour to include this in a future release.

Is this feature close to release? Would be a really exciting addition. Thanks!

**Comment by deejay-hub** [comment, marked as answer]
The feature is in the Procyon Pull Request (https://github.com/deejay-hub/timeline-lwc/pull/190).

We aim to publish the next release v1.12 (Ross) in the next 2 months. Procyon is planned for sometime after that. Since it's an open source project it's hard to be more specific than that. If you're interested feel free to try it out, test, fix bugs and send pull requests to help!
